coffeecard/CoffeeCard.Library/Migrations/20230714211716_PurchaseOrderIdUniqueAndExternalTransactionId.cs

```diff
@@ -0,0 +1,65 @@
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace CoffeeCard.Library.Migrations
+{
+    public partial class PurchaseOrderIdUniqueAndExternalTransactionId : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            // Make transactionId null, where not MobilePayV1, or MobilepayV2, and generate unique Guid orderIds
+            migrationBuilder.Sql("update Purchases set OrderId = NEWID(), TransactionId=null where Type != 'MobilePayV1' and Type != 'MobilePayV2' or OrderId = 'OLD PURCHASES'");
```

Comment:
Where do we set this?

Reply:
Not sure what you mean? Set is the syntax for updating a specific column in sql, and NEWID() is a build in function of sql server, so we are essentially updating all purchases with a new guid as the orderId where it wasn't already.

Reply:
Sorry. I meant the 'Old purchases'. Do we have entries with this name?

Reply:
I think there were like 2 or 3 of them in the production dataset. I'm guessing they are the result of someone moving some data manually at one point or another. Not something we set from the code